media-libs/libsfml/libsfml-2.4.2-r1.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils versionator
+
+MY_P="SFML-${PV}"
+
+DESCRIPTION="Simple and Fast Multimedia Library (SFML)"
+HOMEPAGE="http://www.sfml-dev.org/ https://github.com/SFML/SFML"
+SRC_URI="https://github.com/SFML/SFML/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="ZLIB"
+SLOT="0/$(get_version_component_range 1-2)"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug doc examples"
+
+RDEPEND="
+	media-libs/flac
+	media-libs/freetype:2
+	media-libs/libpng:0=
+	media-libs/libogg
+	media-libs/libvorbis
+	media-libs/mesa
+	media-libs/openal
+	sys-libs/zlib
+	virtual/jpeg:0
+	kernel_linux? (
+		virtual/libudev:0
+	)
+	virtual/opengl
+	!kernel_Winnt? (
+		x11-libs/libX11
+		x11-libs/libXrandr
+		x11-libs/libxcb
+		x11-libs/xcb-util-image
+	)
+"
+DEPEND="
+	${RDEPEND}
+	doc? ( app-doc/doxygen )
+"
+
+DOCS=( changelog.txt readme.txt )
+
+PATCHES=( "${FILESDIR}"/${PN}-2.2-no-docs.patch )
+
+S="${WORKDIR}/${MY_P}"
+
+src_prepare() {
+	# Do not install bundled deps; only applicable on mingw builds and maybe osx.
+	# Upstream PR: https://github.com/SFML/SFML/pull/1237
+	epatch "${FILESDIR}"/${PN}-2.4.2-no-install-extlibs-mingw.patch
+
+	sed -i "s:DESTINATION .*:DESTINATION /usr/share/doc/${PF}:" \
+		doc/CMakeLists.txt || die
+
+	default
```

Comment:
never use `default` with cmake, always run `cmake-utils_src_prepare`

Reply:
Ah, I can't believe I didn't see this last year. Will fix.